src/codegen.zig

```diff
@@ -57,7 +57,7 @@ fn importBackend(comptime backend: std.builtin.CompilerBackend) type {
         .stage2_powerpc => unreachable,
         .stage2_riscv64 => @import("arch/riscv64/CodeGen.zig"),
         .stage2_sparc64 => @import("arch/sparc64/CodeGen.zig"),
-        .stage2_spirv => @import("codegen/spirv.zig"),
+        .stage2_spirv => @import("arch/spirv/CodeGen.zig"),
```

Comment:
Backends should be in `src/codegen` going forward, so the old location was fine.

Reply:
Can you elaborate more on the difference? I thought `src/arch` could be a better place since `CodeGen.zig` (old `spirv.zig`) is an struct now.